VERSION

```diff
@@ -1 +1 @@
-1.1.01
+1.1.02
```

Comment:
And actually I'm an idiot - we don't need to bump the version if there aren't changes to the server. Sorry for missing this - I had just woken up :P